docs/pages/versions/unversioned/distribution/security.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Security
+---
+
+Before you enter outside credentials or provide other sensitive data to a third party tool you should ask yourself whether you trust the tool to use it responsibly and protect it. Due to the native of what goes in to building an app binary for distribution on app stores, the Expo standalone app build service requires various pieces of information with varying degrees of sensitivty. The purpose of this document is to explain what those are, how we store them, and what could go wrong if they were to be compromised.
+
+Most data stored by Expo — credentials or otherwise — is encrypted at rest by our cloud provider, Google Cloud. Credentials are additionally encrypted using [KMS](https://cloud.google.com/kms/). Credentials are only unencrypted for as long as we need them in memory in the standalone app builders or notification services. Credentials are always encrypted in our databases, message queues and other less transient parts of the system.
+
+All of the files and strings related to the information explained below can be downloaded and removed from Expo servers (if it is stored there at all in the first place), and some of it may be available through other locations such as the Apple Developer portal.
+
+## iOS Push Notification credentials
+
+There are two types of iOS push notification credentials, one modern recommended approach recommended by Apple and the legacy approach. The default behavior is to use the modern approach, but users may opt-in to the legacy approach by providing a p12 certificate.
+
+### Modern: push key (p8) + key ID (string)
+
+Each developer account has up to two push keys, each of which can send notifications to any app on the account.
+
+Push keys are revocable from the Apple Developer Center. If you revoke them, notifications will stop working. If you provision a new push key and upload it to Expo then notifications will resume working. Device tokens are not invalidated when push credentials are revoked.
+
+### Legacy: push certificate (p12) + password (string)
+
+Each app has a different push certificate and a certificate lets you send notifications to the app associated with the certificate.
+
+Push certificates are revocable from the Apple Developer Center. If you revoke them, notifications will stop working. If you provision a new certificate and upload it to Expo then notifications will resume working. Device tokens are not invalidated when push credentials are revoked.
+
+### Consequences if compromised
+
+If a malicious actor were to somehow gain access to the credentials, they would be able to send push notifications to your app. However, they would need to know which device tokens to send them too.
+
+### Consequences if lost
+
+None, they are available through the Apple Developer console. You can easily create new ones too.
```

Comment:
```suggestion
The Apple Developer console lets you download an APNs Auth Key only when it is created. If an Auth Key is lost, it can be revoked through the Apple Developer console and replaced with a new key.

You can download your push certificates anytime from the Apple Developer console if they are lost.
```